### PR TITLE
Check box event

### PR DESCRIPTION
**Pull request by jonkiky**

### CODE REVIEW

File: src/bento/dashboardTabData.js [hunk 1]
```diff
@@ -1022,3 +1022,34 @@ query fileOverview($file_uuids: [String], $offset: Int = 0, $first: Int = 100000
...
+)
+{
+    fileIdsFromFileName(
+        file_name:$file_name, 
+        offset:$offset,
+        first:$first,
+        order_by:$order_by
+    )
+    {
+        file_name
```
Comment (AjayDoddapaneni) [inline src/bento/dashboardTabData.js:1041]: we don't need the file name

File: src/bento/dashboardTabData.js [hunk 2]
```diff
@@ -1022,3 +1022,34 @@ query fileOverview($file_uuids: [String], $offset: Int = 0, $first: Int = 100000
...
+        file_uuid
+    }
+
+    fileIdsFromFileNameDesc(
```
Comment (AjayDoddapaneni) [inline src/bento/dashboardTabData.js:1045]: one fileIdsFromFileName should be sufficent

Comment (jonkiky) [reply src/bento/dashboardTabData.js:1034]: updated

File: src/pages/fileCentricCart/cartController.js [hunk 3]
```diff
@@ -1,3 +1,4 @@
+/* eslint-disable */
```
Comment (AjayDoddapaneni) [inline src/pages/fileCentricCart/cartController.js:1]: this can goo!!!

File: src/pages/dashboardTab/store/dashboardReducer.js [hunk 4]
```diff
@@ -351,6 +358,86 @@ export async function fetchAllFileIDsForSelectAll(fileCount = 100000) {
...
+      });
+  } else {
+  // if in sample or case tab, get file id from backend
+    const SELECT_ALL_QUERY = getState().currentActiveTab === tabIndex[1].title
+      ? GET_ALL_FILEIDS_SAMPLESTAB_FOR_SELECT_ALL
+      : GET_ALL_FILEIDS_CASESTAB_FOR_SELECT_ALL;
+
+    let caseIds = [];
+    let sampleIds = [];
+    if (getState().currentActiveTab === tabIndex[1].title) {
```
Comment (AjayDoddapaneni) [inline src/pages/dashboardTab/store/dashboardReducer.js:396]: Looks like are using getState().currentActiveTab multiple places and multiple times in this function can you declare it as contact at starting of function?

**Pull request closed without merging**